code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -93,6 +94,8 @@
 
 /obj/item/weapon/card/id/examine(mob/user)
 	..()
+	if(registered_race && registered_race.id != "human")

```

Comment:
What about the thirty-odd blank ID cards that spawn on the station? What assigned species do they have?